express-sendfile/index.js

```diff
@@ -0,0 +1,19 @@
+const express = require('./node_modules/express');
+
+const app = express();
+
+app.use(function (req, res, next) {
+  const options = {
+    root: __dirname
+  };
+  res.sendFile('./hello.txt', options, function (err) {
+    if (err) {
+      next(err);
+    }
+  });
```

Comment:
The server should also be able to send the HTML, CSS, or JavaScript file if requested.